workspace/test_configurations.go

```diff
@@ -62,8 +62,7 @@ func (c *TestConfiguration) GetTestCommand() (string, error) {
 // some tracks aren't (or won't be) implemented; every track is listed either way
 var TestConfigurations = map[string]TestConfiguration{
 	"8th": {
-		Command:        "bash tester.sh",
-		WindowsCommand: "tester.bat",
+		Command: "8th test.8th",
```

Comment:
Looking at the 8th command documentation I see `-f` is optional when executed against a single file so I understand it is not needed.  But the Exercism documentation references the file flag `-f` in the test command. Should it be added for consistency?
```suggestion
		Command: "8th -f test.8th",
```